src/test/incremental/change_symbol_export_status.rs

```diff
@@ -2,7 +2,7 @@
 // compile-flags: -Zquery-dep-graph
 
 #![feature(rustc_attrs)]
-#![rustc_partition_codegened(module = "change_symbol_export_status-mod1", cfg = "rpass2")]
+#![rustc_partition_reused(module = "change_symbol_export_status-mod1", cfg = "rpass2")]
```

Comment:
This is wrong, but preexisting.

Reply:
I'm not sure what this change even means.